Amplify/Categories/Predictions/Models/AgeRange.swift

```diff
@@ -7,6 +7,7 @@
 
 import Foundation
 
+/// Struct describing age range (low, high)
```

Comment:
General thoughts:

- "Struct" is redundant; it's easy enough to see what the actual concrete thing is. 
- These comments would be most useful if they described their use cases. Where do I as a customer have to pass them? Where should I as a customer expect to see them? E.g., if this was the return value from a predictions call: e.g., "Age range to filter an identifyFaces search" or "Age range of a detected entity/face/whatever identified in a identifyFaces call." (These aren't real calls, just examples of how the doc could be more useful to customers.)